src/lib/y2storage/disk.rb

```diff
@@ -68,6 +68,13 @@ def usb?
       transport.to_sym == :usb
     end
 
+    # Checks if it's in network
+    #
+    # @return [Boolean]
+    def network_transport?
+      transport.network?
+    end
```

Comment:
Having a method `network_transport?` that is implemented as `transport.network?` is weird. I would say the problem is that the name of the method reveals too much about the logic behind it. Don't you think a better name would be simply `in_network?`. Most of the users of the `Disk` class don't even know the "transport" concept.

As a plus, `Disk#in_network?` is more consistent with `Filesystems::Base#in_network?`